src/providers/dataProvider/index.ts

```diff
@@ -135,8 +135,17 @@ export const dataProvider = (apiUrl: string): DataProvider => ({
   },
 
```

Comment:
ok, for next week.  Let's define these as a const array somewhere in this file. Call it "Tables_with_string_PK".

We should probably use this code for all `Many` verbs.

Reply:
We don't need that anymore as the issue in `rco-soul` for string `ids` has been resolved